### PR TITLE
Implement feedback screen

### LINKED ISSUE #5395: Add feedback ui in HTML5 client like in Fash client

### PR DESCRIPTION
**Pull request by Tainan404**
Adds a modal for when the user leaves the client, asking for evaluation and improvement tips.

PS: is with a temporary icon, until the star icon is added to the BBB font.
Closes #5395.

### CODE REVIEW

File: bigbluebutton-html5/imports/ui/components/feedback-screen/component.jsx [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+
+
+const intlMessages = defineMessages({
+  title: {
+    id: 'app.feedback.title',
+    description: 'Join mic audio button label',
+  },
+  subtitle: {
+    id: 'app.feedback.subtitle',
+    description: 'Join mic audio button label',
```
Comment (antobinary) [inline bigbluebutton-html5/imports/ui/components/feedback-screen/component.jsx]: please update description

**Comment by KDSBrowne** [comment]
The translated strings are lost when you reload the page on the feedback-screen.

image

Also, instead of the text input being disabled; If i'm not mistaken, it shouldn't be rendered until one of the star's are selected.

feedback-modal-flash

**Comment by KDSBrowne** [comment]
image

**Comment by KDSBrowne** [comment]
This console error is shown when the feedback screen is first rendered.
image

**Pull request merged**